File: pset1/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float dollars;
    do
    {
        dollars = get_float("Change owed: ");
    }
    while (dollars < 0);
    int coins = round(dollars * 100);
    int total = 0;

    for (; coins > 24; coins -= 25)
    {
        total += 1;
    }
    for (; coins > 9; coins -= 10)
    {
        total += 1;
    }
    for (; coins > 4; coins -= 5)
    {
        total += 1;
    }
    for (; coins > 0; coins -= 1)
    {
        total += 1;
    }
    printf("%d\n", total);
}

File: pset6/cash.py
from cs50 import get_float 

while True:
    dollars = get_float("Change owed: ")
    if dollars >= 0:
        break
coins = round(dollars * 100)
total = 0

while coins > 24:
    total += 1
    coins -= 25

while coins > 9:
    total += 1
    coins -= 10

while coins > 4:
    total += 1
    coins -= 5

while coins > 0:
    total += 1
    coins -= 1
    
print(total)

